BUCK

```diff
@@ -2,7 +2,7 @@
 genrule(
     name = '_bundle-javascript',
     srcs = glob( [ 'src/main/javascript/**/*' ] ),
-    cmd = 'browserify --debug src/main/javascript/home.js > $OUT',
+    cmd = 'nodejs ./../../../node_modules/.bin/browserify --debug src/main/javascript/home.js -o $OUT',
```

Comment:
I think it's only debian packages that call node "nodejs" so what about we make this "node" for better compatibility?